src/v2/coins/abstractUtxoCoin.js

```diff
@@ -768,11 +769,11 @@ class AbstractUtxoCoin extends BaseCoin {
     let decompiledSigScript = bitcoin.script.decompile(signatureScript);
 
     const isSegwitInput = currentInput.witness.length > 0;
-    const isBech32Input = isSegwitInput && (signatureScript.length === 0);
     if (isSegwitInput) {
+      const isNativeSegwit = signatureScript.length === 0;
```

Comment:
Did not change it here to make it consistent with `isSegwitInput`. We could make `isSegwitInput` something like `isP2shP2wshOrP2wshInput` but that seems unnecessarily wordy and probably more confusing.

Reply:
Sounds reasonable

Reply:
Yeah, that sounds reasonable.